includes/class-sensei-course-progress-widget.php

```diff
@@ -251,15 +252,19 @@ function widget( $args, $instance ) {
 						}
 
 						?>
-						<li class="course-progress-module"><h3 class="module-title"><?php echo $module_title; ?></h3></li>
+						<li class="course-progress-module">
+							<h3 class="module-title">
+								<?php echo $module_title; // phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped ?>
```

Comment:
To be extra safe we usually run anything that's supposed to be unescaped HTML through wp_kses_post.